torchvision/transforms/functional_tensor.py

```diff
@@ -228,13 +310,11 @@ def adjust_gamma(img: Tensor, gamma: float, gain: float = 1) -> Tensor:
     result = img
     dtype = img.dtype
     if not torch.is_floating_point(img):
-        result = result / 255.0
+        result = convert_image_dtype(result, torch.float32)
```

Comment:
unfortunately torch.get_default_dtype() isn't scriptable either.

Reply:
It's fine to default to `float32` for all computations, I think that using `get_default_dtype()` could lead to surprises if the user switched the defaults